src/components/Match/TeamTable.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { isRadiant } from 'utility';
+import strings from 'lang';
+import Heading from 'components/Heading';
+import { IconRadiant, IconDire } from 'components/Icons';
+import Table from 'components/Table';
+import PicksBans from './Overview/PicksBans';
+import styles from './Match.css';
+
+const filterMatchPlayers = (players, team = '') => players.filter(player => (
+  (team === 'radiant' && isRadiant(player.player_slot)) || (team === 'dire' && !isRadiant(player.player_slot)) || team === ''),
+).sort((a, b) => a.player_slot - b.player_slot);
+
+export default ({
```

Comment:
TeamTable now has a new function signature, it accepts the player array only (not match object)